tracing-subscriber/src/lib.rs

```diff
@@ -107,7 +107,7 @@ pub(crate) mod thread;
 
 #[cfg(feature = "env-filter")]
 #[allow(deprecated)]
```

Comment:
should be able to remove this `allow(deprecated)`

Reply:
(we should probably check for any other `allow(deprecated)`s?)

Reply:
fixed!